### PR TITLE
Add seccomp notifier blog post

### PR DESCRIPTION
**Pull request by saschagrunert**
Adding a blog post about how to use the seccomp notifier for profile debugging.

A possible publishing date has to be defined as well.

### CODE REVIEW

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+Debugging software in production is one of the biggest challenges we have to
+face in our containerized environments. Being able to understand the impact of
+the available security options, especially when it comes to configuring our
+deployments, is one of the key aspects to make the default security in
+Kubernetes stronger. We have all those logging, tracing and metrics data already
+at hand, but how do we assemble the information they provide into something
+human readable and actionable?
+
+[Seccomp][seccomp] is one of the standard mechanisms to protect a [Linux][linux]
+based Kubernetes application from malicious actions by interfering with its
```
Comment (rphillips) [inline content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:21]: interfering could be clearer with 'blocking'?

Comment (saschagrunert) [reply content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:21]: This depends on the action, we may also just log them or allow.

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+  "syscalls": [
+    {
+      "names": ["chmod", "chown", "open", "write"],
+      "action": "SCMP_ACT_ALLOW"
+    }
+  ]
+}
+```
+
+The above profile errors per default by specifying the `defaultAction` of
```
[the remark this answers is not in the document]
Comment (saschagrunert) [reply content/en/blog/_posts/2022-11-XX-seccomp-notifier.md]: Fixed :+1:

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+
+Debugging software in production is one of the biggest challenges we have to
+face in our containerized environments. Being able to understand the impact of
+the available security options, especially when it comes to configuring our
+deployments, is one of the key aspects to make the default security in
+Kubernetes stronger. We have all those logging, tracing and metrics data already
+at hand, but how do we assemble the information they provide into something
+human readable and actionable?
+
+[Seccomp][seccomp] is one of the standard mechanisms to protect a [Linux][linux]
```
Comment (sftim) [inline content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:18]: No need to hyperlink to Linux; we can assume readers have either heard of it or know of Google.

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+
+We can also test if the web server itself works as intended:
+
+```shell
+> curl 10.85.0.3
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+…
```
Comment (sftim) [inline content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:228]: This isn't a valid shell script. I recommend changing the format indicator on the previous line.

Imagine that we want one day to lint these scripts. We probably don't, but we do want to be able to change the highlighter without checking all of the pages that might misuse it.

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md (hunk 3 above)
Comment (saschagrunert) [reply content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:18]: Removed it as suggested

File: content/en/blog/_posts/2022-11-XX-seccomp-notifier.md (hunk 4 above)
Comment (saschagrunert) [reply content/en/blog/_posts/2022-11-XX-seccomp-notifier.md:227]: Alright, I changed it to `console`.

**Comment by saschagrunert** [comment]
@sftim would this article qualify for the next week?

**Comment by sftim** [comment]
How about Friday, the 2nd of December? We (blog team) might get this out by then.

/hold

Publication date needs to go in to Markdown and into the paths of the source files.

**Comment by sftim** [comment]
@saschagrunert could you get in touch with SIG Security about a technical review for this?

**Comment by saschagrunert** [comment]
Changed the date to match Dec 2.

I'm asking in the SIG Security slack for review, but also would have another pair of eyes from @haircommander.

**Comment by haircommander** [comment]
/lgtm

great post!

**Comment by sftim** [comment]
I meant to hold this one. Oops. There's still a day or so to yank it if need be.

**Comment by haircommander** [comment]
I am happy to say so!

**Comment by saschagrunert** [comment]
All good, thank you!

**Pull request merged**